infrastructure/cdn-in-a-box/traffic_ops/Dockerfile

```diff
@@ -79,6 +79,9 @@ ADD traffic_router/core/src/test/resources/geo/GeoLite2-City.mmdb.gz /opt/traffi
 
 WORKDIR /opt/traffic_ops/app
 ADD traffic_ops/app/cpanfile traffic_ops/install/bin/install_goose.sh ./
+
+# Start with the existing traffic_ops/app/local directory
+COPY traffic_ops/app/local local
```

Comment:
Wouldn't this try to copy my macOS-based `local` dir into a centOS container?

Reply:
It will, but `carton` replaces anything it cannot use.

Reply:
Granted, I haven't tested on macOS yet. Doing that now

Reply:
Tested on Mac. If the wrong versions of CPAN modules or their dependencies are installed, Carton installs the correct version for the platform (since we don't track a `cpanfile.snapshot`). Architecture-specific components are stored in `./local/lib/perl5/darwin-thread-multi-2level` for my Mac, in `./local/lib/perl5/x86_64-linux-thread-multi` for CDN in a Box, etc., so builds for more than one architecture do not conflict.

The only catch is that CPANminus does not have those checks. Installing Carton in the system-wide Perl library in e8ea729ce1. Works for me when CPAN dependencies are vendored locally on macOS using default Perl, v5.18.4, and on Linux.

Reply:
I was mistaken, `x86_64-linux-thread-multi` is not specific enough. Using CPAN modules built with CentOS 7 on CentOS 8, I get

```
/usr/bin/perl: symbol lookup error: /opt/traffic_ops/app/local/lib/perl5/x86_64-linux-thread-multi/auto/List/Util/Util.so: undefined symbol: Perl_xs_apiversion_bootcheck
```

Opened #5262 to cache modules by OS version and to use a `local` directory specific to CDN-in-a-Box.